packages/plugin-flex/src/commands/flex/plugins/archive/configuration.ts

```diff
@@ -0,0 +1,42 @@
+import { Configuration } from 'flex-plugins-api-toolkit';
+import { OutputFlags } from '@oclif/parser/lib/parse';
+
+import * as flags from '../../../../utils/flags';
+import ArchiveResource from '../../../../sub-commands/archive-resource';
+import { createDescription } from '../../../../utils/general';
+import { archiveConfiguration as archiveConfigurationDocs } from '../../../../commandDocs.json';
+import FlexPlugin from '../../../../sub-commands/flex-plugin';
+
+export default class FlexPluginsArchiveConfiguration extends ArchiveResource<Configuration> {
+  static description = createDescription(archiveConfigurationDocs.description, false);
+
+  static flags = {
+    ...FlexPlugin.flags,
+    sid: flags.string({
+      description: archiveConfigurationDocs.flags.sid,
+      required: true,
+    }),
+  };
+
+  /**
+   * @override
+   */
+  async doArchive(): Promise<Configuration> {
+    return this.pluginsApiToolkit.archiveConfiguration({ sid: this._flags.sid });
+  }
+
+  /**
+   * @override
+   */
+  getName(): string {
+    return `Configuration ${this._flags.sid}`;
```

Comment:
are you not using name here because it's not unique? (are config names unique, i forget)

Reply:
ya, `name` is not unique. But good idea, I'll say `Configuration NAME(SID)` as both - most probably do make the names unique

Reply:
Oh wait, we don't have the `name` actually - it is being passed via a flag. So ya can't use name here.

Reply:
ah okay